public/app/features/dashboard/dashboard_model.ts

```diff
@@ -146,6 +152,18 @@ export class DashboardModel {
       list: _.map(this.templating.list, variable => (variable.getSaveModel ? variable.getSaveModel() : variable)),
     };
 
+    if (!saveVariables && copy.templating.list.length === this.originalTemplating.length) {
+      for (let i = 0; i < copy.templating.list.length; i++) {
```

Comment:
Is the loop really needed? I'll expect that the following don't work for some reason, or?
```javascript
copy.templating.list = this.originalTemplating;
```

Reply:
this.originalTemplating only consists of name and current since it should only check the current value.